data/json/monsters/fish.json

```diff
@@ -484,6 +498,7 @@
   },
   {
     "id": "mon_fish_crayfish",
+    "looks_like": "mon_fish_trout",
```

Comment:
I think lobster and crawfish are different enough from fish to merit their own tile.
Probably make one look_like the other, no opinion which one.